operatorhub/subscription.yml

```diff
@@ -0,0 +1,38 @@
+---
+apiVersion: v1
+kind: Template
+message: |-
+  The following subscription has been created: ${NAME}
+metadata:
+  name: subscription-template
+objects:
+- apiVersion: operators.coreos.com/v1alpha1
+  kind: Subscription
+  metadata:
+    name: "${NAME}"
+    namespace: "${OPERATOR_NAMESPACE}"
```

Comment:
I think this line should be removed and instead set the namespace from cli or from the applier.